match_minima.py

```diff
@@ -0,0 +1,787 @@
+#!/usr/bin/env python
+
+"""
+match_minima.py
+
+Match conformers from sets of different optimizations.
+Compute relative energies of corresponding conformers to a reference conformer.
+Generate plots for relative conformer energies (one plot per mol).
+
+By:      Victoria T. Lim
+Version: Nov 18 2019
+
+"""
+
+import os
+import sys
+import numpy as np
+import pickle
+import itertools
+import copy
+import collections
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import openeye.oechem as oechem
+
+### ------------------- Functions -------------------
+
+
+def read_mols(infile, mol_slice=None):
+    """
+    Open a molecule file and return molecules and conformers as OEMols.
+    Provide option to slice the mols to return only a chunk from the
+    specified indices.
+
+    Parameters
+    ----------
+    infile : string
+        name of input file with molecules
+    mol_slice : list
```

Comment:
It would be good to say `mol_slice : list[int]` here, to clarify the data type in the list